src/borg/testsuite/helpers.py

```diff
@@ -579,7 +579,7 @@ def test_file_size_precision():
 
 def test_file_size_sign():
     si_size_map = {
-        0: '0 B',
+        0: '+0 B',
```

Comment:
hmm, when used in a difference context (like: how much bytes do we have additionally / less?) having a "neutral" 0 would be nice.

Reply:
Why would you force signs then? Minus sign is always displayed. I changed to the default Python formater sign handling.

Reply:
so, the test should not be changed.

Reply:
How can one force that any sign is always displayed then? This would not be possible. IMHO, it makes more sense to have the following two possibilities:
```
-1
0
1
```
and
```
-1
+0
+1
```

Reply:
Obviously, the sign=True param was never intended to display a sign for 0, see the unit test.

And I already explained why: this is about difference.

E.g. if something got bigger, we want to display +123MB.
If something got small, we want to display -456MB.
But if it is the same, we want just a plain 0 - no plus, no minus in that case.

Reply:
see my other comment.

Reply:
I do not intend to deviate from the default python and mathematical convention. You can finish the PR according to your needs.

Reply:
OK, I'll finish it. Thanks!